### DISCUSSION TITLE
for changing the education pack?

### DISCUSSION THREAD

**Discussion by KhemSaud** [community/community#135815] [GitHub Education] [answered]
### Hi everyone,

I was about to choose free GITHUB pro but mistakely i choose student developer pack . i look over student developer pack i didnt find any thing benifical to me . Can change my pack from STUDENT DEVELOPER PACK to FREE GITHUB PRO? . i am just the beggineer at this platform.  Your every suggestion will be appericiated and valued.

**Comment by TahaBakhtari** [comment, marked as answer]
If you picked the GitHub Student Developer Pack by mistake and want to change to the free GitHub Pro plan here's what you should do:

1. Know Your Choices:
The Student Developer Pack has lots of free tools and resources for students. GitHub Pro gives you more advanced GitHub features.
2. Get in Touch with GitHub Support:
Visit the GitHub Support page.
Tell them you're new to this and chose the Student Developer Pack by accident.
Ask them to help you switch to the free GitHub Pro plan.
3. Be Patient:
GitHub Support will tell you how to change your plan.
This way, you'll end up with the plan that works best for you.

**Comment by davevad93** [comment]
Hi @KhemSaud ,

The `Student Developer Pack` offers a free subscription to `GitHub Pro` among other things, there's no other way to get GitHub Pro for free so you don't have to change anything. I suggest you check out the [pack offers](https://education.github.com/pack/offers) because there are a lot of interesting things you could get for free, like a free subscription to `Copilot`, free access to `Microsoft Azure` and a bunch of learning platforms.

Please note that student validation lasts for 2 years, after which you will have to re-verify your student status by submitting your documentation again on the application form.
